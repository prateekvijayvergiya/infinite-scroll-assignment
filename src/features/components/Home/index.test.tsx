import Component from '.'
import makeSnapshot from '../../testing/makeSnapshot'

jest.mock('@material-ui/core/Grid', () => 'Grid')
jest.mock('@material-ui/core/Typography', () => 'Typography')
jest.mock('@material-ui/core/Button', () => 'Button')
jest.mock('./usersList/index', () => 'UsersList')
jest.mock('../common/spacer', () => 'Spacer')

describe('Home', () => {
  it('renders correctly', () => {
    makeSnapshot(<Component />)
  })
})
