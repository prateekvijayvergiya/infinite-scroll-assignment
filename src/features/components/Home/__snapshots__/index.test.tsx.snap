// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<Grid
  alignItems="center"
  container={true}
  direction="column"
  justify="center"
>
  <Grid
    item={true}
  >
    <Typography
      align="center"
      gutterBottom={true}
      variant="h4"
    >
      Find your perfect match
    </Typography>
  </Grid>
  <Grid
    item={true}
  >
    <Button
      color="primary"
      onClick={[Function]}
      variant="contained"
    >
      Logout
    </Button>
  </Grid>
  <Grid
    item={true}
  >
    <Spacer />
  </Grid>
  <Grid
    item={true}
  >
    <UsersList />
  </Grid>
</Grid>
`;
